docs/resources/cluster.md

```diff
@@ -259,6 +259,16 @@ cluster_log_conf {
 }
 ```
 
+Example of pushing all cluster logs to UC Volumes:
+
+```hcl
+cluster_log_conf {
+  volumes {
```

Comment:
Where is `volumes` and the allowed arguments to it actually documented in `cluster_log_conf`?

Reply:
it's part of Go SDK now, and rollout this week